src/Microsoft.ML.OnnxConverter/OnnxContextImpl.cs

```diff
@@ -6,8 +6,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using Microsoft.Data.DataView;
-using Microsoft.ML.Model.OnnxConverter;
-
+using Microsoft.ML.Runtime;
```

Comment:
That it removed the spacing here for whatever reason is a little annoying but just a completely pointless nit. (Certainly the good I've observed so far far outweighs this one little bad.)